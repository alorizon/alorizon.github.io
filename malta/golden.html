<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8" />
		<link rel="stylesheet" href="../ville_12.css" />
		<link rel="stylesheet" href="../ville.css" />
		<link rel="stylesheet" media="screen and (max-device-width: 640px)" href="../ville_smartphone.css" />
		<link rel="icon" type="image/png" href="https://res.cloudinary.com/lor/image/upload/v1606159285/favicon_rm0eyd.jpg"/>
		<title>golden</title>
	</head>
	
	<body>

		<a href="../index.html"><img id="logo" src="https://res.cloudinary.com/lor/image/upload/v1606159285/favicon_rm0eyd.jpg"></a>
		<a href="malta.html"><img id="back" src="https://res.cloudinary.com/lor/image/upload/v1608570668/malta_flagg_loc2il.jpg"></a>

		<br/><br/>

		<h1><a href="malta.html">Malta</h1></a>

		<h2>Golden Bay</h2>
									
				<p>	Golden Bay, Ramla tal-Mixquqa en maltais, située au nord de l'île de Malte, est la plus grande plage du pays. J'ai eu super froid ce jour là et le vent était très présent, je n'ai donc, encore une fois, pas eu le courage de me baigner. 
				<br/><br/>
				Après cette vue magnifique, je me suis lancée dans une randonnée en direction de Popeye Village. 
				</p>

				<div class=container>
					<div class="gallery">

						<figure class="gallery__item gallery__item--1 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667629/malta/golden/popeye2_be7xfb.jpg" class="gallery__img" alt=""><div class="text"><em>POPEYE VILLAGE</em><br/><strong>- 1979 -</strong><br/><br/>La construction du village commence en juin 1979. Une équipe de 165 personnes travaille pendant sept mois à la construction des dix-neuf bâtiments de bois qui composent le village. <br/><br/>
							Le tournage débute le 23 janvier 1980 et se termine le 19 juin de la même année. Bien que le film n'ait pas eu un franc succès, c'est une attraction touristique populaire.</div>
						</figure>
						<figure class="gallery__item gallery__item--5 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667630/malta/golden/popeye2h_frkebl.jpg" class="gallery__img" alt=""><div class="text"><em>POPEYE VILLAGE</em><br/><strong>- 1979 -</strong></div>
						</figure>
						<figure class="gallery__item gallery__item--2 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667586/malta/golden/vueh_snwbbv.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--3 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667566/malta/golden/golden2_t8iu7h.jpg" class="gallery__img" alt=""><div class="text"><em>GOLDEN BAY</em></div>
						</figure>
						<figure class="gallery__item gallery__item--6 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667608/malta/golden/mdina2h_ous3m6.jpg" class="gallery__img" alt=""><div class="text"><em>L-IMDINA</em><br/><br/>Mdina est la moins peuplée des localités de Malte avec environ 300 habitants, d'où son surnom de The Silent City (la cité silencieuse). 
							<br/><br/></div>
						</figure>
						<figure class="gallery__item gallery__item--12 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667619/malta/golden/moi_tcjkdf.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--9 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667546/malta/golden/mdina2_wpbktl.jpg" class="gallery__img" alt=""><div class="text"><em>L-IMDINA</em></div>
						</figure>
						<figure class="gallery__item gallery__item--8 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667587/malta/golden/cathedrale_paul_uwbhft.jpg" class="gallery__img" alt=""><div class="text"><em>IL-KATIDRAL METROPOLITAN TA' SAN PAWL</em><br/><strong>- 1697-1702 -</strong><br/><br/> La Cathédrale St Paul a été conçue par l’architecte Lorenzo Gafa. Elle est construite entre 1697 et 1702 pour remplacer l’ancienne cathédrale de style normand, détruite par le tremblement de terre du 11 janvier 1693. 
							<br/><br/></div>
						</figure>
						<figure class="gallery__item gallery__item--10 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667635/malta/golden/popeye3h_ii8mge.jpg" class="gallery__img" alt=""><div class="text"><em>POPEYE VILLAGE</em><br/><strong>- 1979 -</strong></div>
						</figure>
						<figure class="gallery__item gallery__item--4 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667548/malta/golden/tour_zachvw.jpg" class="gallery__img" alt=""><div class="text"><em>TOUR GĦAJN TUFFIEĦA</em><br/><br/>Ancienne tour de guet construite par les Chevaliers de Saint-Jean.</div>
						</figure>
						<figure class="gallery__item gallery__item--11 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667571/malta/golden/popeye4h_fcpzfk.jpg" class="gallery__img" alt=""><div class="text"><em>POPEYE VILLAGE</em><br/><strong>- 1979 -</strong></div>
						</figure>
						<figure class="gallery__item gallery__item--7 caption">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606667581/malta/golden/pastizzi_ggoguz.jpg" class="gallery__img" alt=""><div class="text"><em>PASTIZZI</em></div>
						</figure>
											
						</div>
					</div>

				<p>
				Ayant visité tout ça en une grosse matinée (jusqu'à 14h), je me suis ensuite dirigée vers Mdina, L-Imdina en maltais, située dans le centre de Malte. 
				<br/><br/>
				Ayant très très faim après la randonnée du matin et le trajet en bus (avec changements et retards comme d'habitude), j'ai cherché à goûter les fameux pastizzi de Mdina dans la pastizzeria "Serkin Crystal Palace". J'ai pris un pastizzi ricotta et un aux petits pois. 80 centimes pour les deux et c'était délicieux !
				<br/><br/>	
				Me promener dans les petites rues de cette ville paisible m'a fait du bien. C'était une petite pause sans vent dans mes oreilles. 
				<br/><br/>	
				J'ai terminé la visite de la ville par la jolie Cathédrale St. Pierre et St. Paul. 				
				</p>

				<div class="carte">
				<img src="https://res.cloudinary.com/lor/image/upload/v1606414469/malta/cartes/golden_t8cjlm.jpg"></div>

				<div class="prev_city">
				<a href="marsaxlokk.html"><img src="https://res.cloudinary.com/lor/image/upload/v1608569785/accueil/prev_qcrzxx.jpg" title="Marsaxlokk Bay"></a><br/></div>

				<div class="next_city">
				<a href="blue.html"><img src="https://res.cloudinary.com/lor/image/upload/v1606730910/next_f4djfq.png" title="Blue Grotto"></a><br/></div>
				
				<p class="pied_page">Carnet de Voyage <br/><br/>© 2020 - Laure BELTRAN - All Rights Reserved </a></p>

				
				
			
	</body>
</html>