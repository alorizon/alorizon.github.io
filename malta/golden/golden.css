/***********************reset css**********************************/
html, body, div, span, applet, object, iframe,
h1, h2, h3, h4, h5, h6, p, blockquote, pre,
a, abbr, acronym, address, big, cite, code,
del, dfn, em, img, ins, kbd, q, s, samp,
small, strike, strong, sub, sup, tt, var,
b, u, i, center,
dl, dt, dd, ol, ul, li,
fieldset, form, label, legend,
table, caption, tbody, tfoot, thead, tr, th, td,
article, aside, canvas, details, embed, 
figure, figcaption, footer, header, hgroup, 
menu, nav, output, ruby, section, summary,
time, mark, audio, video {
	margin: 0;
	padding: 0;
	border: 0;
	font-size: 100%;
	font: inherit;
	vertical-align: baseline;}
/* HTML5 display-role reset for older browsers */
article, aside, details, figcaption, figure, 
footer, header, hgroup, menu, nav, section {display: block;}
body {line-height: 1;}
ol, ul {list-style: none;}
blockquote, q {quotes: none;}
blockquote:before, blockquote:after,q:before, q:after {content: '';content: none;}
table {border-collapse: collapse;border-spacing: 0;}

/******************Nouvelles polices de caractère*******************/

@font-face 
{font-family : 'prism';
src: url('../../Fonts/prism_regular.otf') format('otf');
src: url('../../Fonts/font.woff2') format('woff2'), 
url('../../Fonts/font.woff') format('woff');}

@font-face 
{font-family : 'old';
src: url('../../Fonts/old_standard.ttf') format('TrueType');}

@font-face 
{font-family : 'poiret';
src: url('../../Fonts/poiret.ttf') format('TrueType');}
/*******************************************************************/

html{ 
  margin:0;
  padding:0;
  background: white; 
  -webkit-background-size: cover; /* pour anciens Chrome et Safari */
  background-size: cover; /* version standardisée */
}

#logo{
 	z-index: 100;
	margin-top: 40px;
	margin-left: 10%;
	height: 40px;
	width: 40px;
	cursor: pointer;}

#back{  
  float: right;
  margin-top: 40px;
  margin-right: 10%;
  height: 40px;
  width: 40px;
  cursor: pointer;}

h1{
	cursor: pointer;
	top: 0px;
	padding: 0px 0px;	
	font-family: 'prism', Verdana, Arial, serif;
	font-size: 400%;
	font-style: normal;
	font-weight: normal;
	text-align: center;
	color: black;}

h2{
	padding: 0px 0px;	
	font-family: 'old', Verdana, Arial, serif;
	font-size: 150%;
	font-style: normal;
	text-align: center;
	color: black;}

 p{
	
	margin-top: 50px;
	padding: 0px 10%;
	font-family: 'old', Verdana, Arial, serif;
	font-size: 100%;
	font-style: normal;
	text-align: justify;
	color: black;}


.container {
  width: 80%;
  margin: 2rem auto; }

.gallery{
  display: grid;
  grid-template-columns: repeat(8, 1.5fr);
  grid-template-rows: repeat(19, 8vw);
  grid-gap: 5px;}

.gallery__img {
    width: 100%;
    height: 100%;
    object-fit: cover;}

.gallery__item--1 { 
    grid-column-start: 1;
    grid-column-end: 5;
    grid-row-start: 1;
    grid-row-end: 4;}

.gallery__item--2 { 
    grid-column-start: 5;
    grid-column-end: 9;
    grid-row-start: 1;
    grid-row-end: 4;}

.gallery__item--3 { 
    grid-column-start: 4;
    grid-column-end: 9;
    grid-row-start: 14;
    grid-row-end: 17;}

.gallery__item--4 { 
    grid-column-start: 4;
    grid-column-end: 9;
    grid-row-start: 4;
    grid-row-end: 7;}

.gallery__item--5 { 
    grid-column-start: 4;
    grid-column-end: 9;
    grid-row-start: 7;
    grid-row-end: 11;}

.gallery__item--6 { 
    grid-column-start: 1;
    grid-column-end: 4;
    grid-row-start: 4;
    grid-row-end: 8;}

.gallery__item--7 { 
    grid-column-start: 1;
    grid-column-end: 5;
    grid-row-start: 11;
    grid-row-end: 14;}

.gallery__item--8 { 
    grid-column-start: 1;
    grid-column-end: 4;
    grid-row-start: 8;
    grid-row-end: 11;}

.gallery__item--9 { 
    grid-column-start: 1;
    grid-column-end: 4;
    grid-row-start: 14;
    grid-row-end: 17;}

.gallery__item--10 { 
    grid-column-start: 5;
    grid-column-end: 9;
    grid-row-start: 11;
    grid-row-end: 14;}

.gallery__item--11 { 
    grid-column-start: 5;
    grid-column-end: 9;
    grid-row-start: 17;
    grid-row-end: 20;}

.gallery__item--12 { 
    grid-column-start: 1;
    grid-column-end: 5;
    grid-row-start: 17;
    grid-row-end: 20;}

.carte img{
  margin-top: 5%;
  margin-left: 15%;
  text-align: center;
  width: 70%;
  height: 70%;}

p.email{
  background-color: #202020;
  padding:30px 0px; 
  font-family: 'poiret', Verdana, serif;
  font-weight: normal;
  font-size: 14px;
  margin-top: 100px;
  margin-bottom: 0%;
  color: white;
  text-align: center;
  height: 30;
  line-height: 15px;}

p.email a, a:active{
  color: white;
  text-decoration: none;}