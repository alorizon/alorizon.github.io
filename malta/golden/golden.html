<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8" />
		<link rel="stylesheet" href="golden.css" />
		<link rel="icon" type="image/png" href="https://res.cloudinary.com/lor/image/upload/v1606159285/favicon_rm0eyd.jpg"/>
		<title>golden</title>
	</head>
	
	<body>

		<a href="../../index.html"><img id="logo" src="https://res.cloudinary.com/lor/image/upload/v1606159285/favicon_rm0eyd.jpg"></a>
		<a href="../malta.html"><img id="back" src="https://res.cloudinary.com/lor/image/upload/v1606159217/back_hd42xc.png"></a>

		<h1>Malta</h1></a>

		<h2>Golden Bay</h2>
									
				<p>	Golden Bay, Ramla tal-Mixquqa en maltais, située au nord de l'île de Malte, est la plus grande plage du pays. J'ai eu super froid ce jour là et le vent était très présent, je n'ai donc encore une fois pas eu le courage de me baigner. 
				<br/><br/>
				Après cette vue magnifique, je me suis lancée dans une randonnée en direction de Popeye Village. 
				<br/><br/>
				La construction du village commence en juin 1979. Une équipe de construction de 165 personnes travaille pendant sept mois à la construction des dix-neuf bâtiments de bois qui composent le village. <br/>
				Le tournage débute le 23 janvier 1980 et se termine le 19 juin de la même année. Bien que le film n'ait pas eu un franc succès, c'est une attraction touristique populaire. 			
				</p>

				<div class=container>
					<div class="gallery">

						<figure class="gallery__item gallery__item--1">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415678/malta/golden/popeye2_t0bx3i.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--2">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415714/malta/golden/popeye2h_qzgaop.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--3">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415319/malta/golden/vueh_fazy7c.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--5">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415361/malta/golden/golden2_ys3cwn.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--4">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415419/malta/golden/mdina2h_kujmsf.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--12">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415450/malta/golden/moi_tztjww.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--7">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415410/malta/golden/mdina2_khjkjq.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--8">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415432/malta/golden/cathedrale_paul_cawozo.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--10">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415725/malta/golden/popeye3h_ktldnd.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--6">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606417284/malta/golden/tour_ibxmpd.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--11">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415771/malta/golden/popeye4h_hnqzr5.jpg" class="gallery__img" alt="">
						</figure>
						<figure class="gallery__item gallery__item--9">
						    <img src="https://res.cloudinary.com/lor/image/upload/v1606415502/malta/golden/pastizzi_ptqlj9.jpg" class="gallery__img" alt="">
						</figure>
											
						</div>
					</div>

				<p>
				Ayant visité tout ça en une grosse matinée (jusqu'à 14h), je me suis dirigée vers Mdina, L-Imdina en maltais, située dans le centre de Malte. 
				<br/>Elle est la moins peuplée des localités de Malte avec environ 300 habitants, d'où son surnom de The Silent City (La cité silencieuse). 
				<br/><br/>
				Ayant très très faim après la randonnée du matin et le trajet en bus (avec changements et retards comme souvent), j'ai cherché à goûter les fameux pastizzi de Mdina dans la pastizzeria "Serkin Crystal Palace". J'ai pris un pastizzi ricotta et un aux petits pois. 80 centimes pour les deux et c'était délicieux !
				<br/><br/>	
				Me promener dans les petites rues de cette ville paisible m'a fait du bien. C'était une petite pause sans vent dans mes oreilles. 
				<br/><br/>	
				J'ai terminé la visite de la ville par la Cathédrale St. Pierre et St. Paul. Conçue par l’architecte Lorenzo Gafa, elle est construite entre 1697 et 1702 pour remplacer l’ancienne cathédrale de style normand, détruite par le tremblement de terre du 11 janvier 1693. 				
				</p>

				<div class="carte">
				<img src="https://res.cloudinary.com/lor/image/upload/v1606414469/malta/cartes/golden_t8cjlm.jpg"></div>
				
				<p class="email">© 2020 - Laure BELTRAN - All Rights Reserved <br/><br/>Si vous souhaitez m'envoyer un mail,<a href="mailto:laurepasquetbeltran@gmail.com"> cliquez ici.</a></p>

				
				
			
	</body>
</html>